homeassistant/components/vicare/sensor.py

```diff
@@ -239,6 +244,7 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         name="Supply Temperature",
         native_unit_of_measurement=TEMP_CELSIUS,
         value_getter=lambda api: api.getSupplyTemperature(),
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Looks like this is missing device class?

Reply:
thanks.
I also added SensorDeviceClass.POWER_FACTOR for Burner modulation. For the other sensors w/o device class I didn't find anything fitting.